### PR TITLE
Wire connect second method

### LINKED ISSUE #242: Update wire section
#### I am submitting a -
* [x] **Feature Request**

#### Describe your bug/feature -
update connecting wires section https://docs.circuitverse.org/#/getting_started?id=connecting-wires
with this new feature 
https://github.com/CircuitVerse/CircuitVerse/pull/1351
#### Are you working on this issue? (Yes/No)
NO

### PR DESCRIPTION
**Pull request by CapTen101**
Fixes #242 

#### Changes done:
- [x] Update the getting started page which now mentions about the **newly created feature** of adding connecting wires with `Ctrl+click` (PR merged-> [#1351](https://github.com/CircuitVerse/CircuitVerse/pull/1351))

**deploy-preview**: https://deploy-preview-252--blissful-einstein-93d693.netlify.app/#/

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add a link from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by CapTen101** [comment]
Kindly review the PR @mahmodHammad @YashKumarVerma

File: docs/getting_started.md [hunk 1]
```diff
@@ -12,12 +12,20 @@ Select the circuit element type and drop on canvas or drag the object onto canva
...
 <video style="width:100%; height: 400px;" controls>
-    <source src="/video/drag.mp4" type="video/mp4">  Sorry, your browser doesn't support embedded videos.
+    <source src="/video/clickwire.mp4" type="video/mp4">  Sorry, your browser doesn't support embedded videos.
 </video>
 
+2. Hold down `Ctrl`, click on a node and simply move the cursor to the final node to which it needs to be attached. Clicking on the wire will insert a node at that position.
 
-Click one node and simply drag the mouse to the other node.
+<video style="width:100%; height: 400px;" controls>
+    <source src="/video/ctrlclickwire.mp4" type="video/mp4">  Sorry, your browser doesn't support embedded videos.
```
Comment (mahmodHammad) [inline docs/getting_started.md:26]: Please create a shorter video.
Show them only the connecting wire mechanism, not putting a components on the circuit.

Also show them that they can quickly put multiple nodes while drawing the wire if they click

Of you didn't get what i mean Please take a look the my pull request for this feature where i have attached both gif for both methods

File: docs/getting_started.md [hunk 2]
```diff
@@ -12,12 +12,20 @@ Select the circuit element type and drop on canvas or drag the object onto canva
 
 ## Connecting wires
 
+There are two ways to connect wires between elements of the circuit:
```
Comment (mahmodHammad) [inline docs/getting_started.md:15]: Could you make it a subheader?
Try putting #### before it

File: docs/getting_started.md [hunk 3]
```diff
@@ -12,12 +12,20 @@ Select the circuit element type and drop on canvas or drag the object onto canva
...
 
+2. Hold down `Ctrl`, click on a node and simply move the cursor to the final node to which it needs to be attached. Clicking on the wire will insert a node at that position.
 
-Click one node and simply drag the mouse to the other node.
+<video style="width:100%; height: 400px;" controls>
+    <source src="/video/ctrlclickwire.mp4" type="video/mp4">  Sorry, your browser doesn't support embedded videos.
+</video>
+Upon reaching the final node, let go of `Ctrl` and simply click on the node to end the wire.
 
 ## Multiple Object Selections
```
Comment (mahmodHammad) [inline docs/getting_started.md:30]: Check why this doesn't work

**Comment by mahmodHammad** [comment]
IMG_20200606_172753

**Comment by CapTen101** [comment]
> IMG_20200606_172753

Yes, I was working on that only. I have fixed it but ran into a problem with squashing the commits. Somehow my head got detached from the latest commit. I'm fixing it. I need little more time with that @mahmodHammad

File: docs/getting_started.md (hunk 2 above)
Comment (CapTen101) [reply docs/getting_started.md:15]: Ok, right :+1:

File: docs/getting_started.md (hunk 3 above)
Comment (CapTen101) [reply docs/getting_started.md:30]: Now it's working. I think the html `video` tag needs one extra line before and after it to identify the markdown written after it.

File: docs/getting_started.md (hunk 1 above)
Comment (CapTen101) [reply docs/getting_started.md:26]: I looked at those gifs in the pull request and they are quite good.

**Comment by CapTen101** [comment]
I've made all changes. I have included short videos. Kindly review them @mahmodHammad :smile:

**Comment by mahmodHammad** [comment]
@CapTen101
All is well, good job🙂.
There's only a small issue, you did it well as intended for the CTRL, but you used a wrong example by connecting 2 inputs together, the ground in digital design represents a 0 logic, in analog design this will work, so please fix that.
You just need to replace the ground with an output element or a LED. .

**Comment by mahmodHammad** [comment]
And please put the components in the circuit before recording to make it shorter

**Comment by CapTen101** [comment]
Oh, ok. Cool👍
I'll insert a new video and will replace the existing ones.

**Comment by CapTen101** [comment]
@mahmodHammad I've added shorter videos with an output element(an LED) as you had suggested. Kindly have a look :slightly_smiling_face:

**Comment by mahmodHammad** [comment]
There's a merge conflict😑, i can not merge, please resolve it

**Comment by mahmodHammad** [comment]
Finally 😂

**Comment by CapTen101** [comment]
Thanks for merging :smile:

**Pull request merged**